README.md

```diff
@@ -1,62 +1,82 @@
 # tethys
 #### ReactiveOps Candidate Technical Challenge
 
-Listed below are a series of tasks. **Complete as many as you can in the allotted time.** While some tasks naturally build on the previous you do not have to complete them in order. You will be asked to provide a public ssh key and you will will be given an username and IP address of an VM on which to complete the tasks. The VM will be running Debian Jessie Linux. **Alotted time is 1Hr.** You may complete the tasks all in one sitting but are not required to do so. Time keeping is up to you. 
+Welcome! 
 
-Before you begin, read the entire list of tasks and conduct any preliminary research required to fully understand the goals. This should not be included in your 1 hour. Reach out with any questions or clarifications you have.
+Listed below are a series of tasks. **Complete as many as you can in the allotted time.** While some tasks naturally build on the previous tasks, you do not have to complete them in order. You will be asked to provide a public ssh key and you will be given an username and IP address of an VM on which to complete the tasks. The VM will be running Debian Jessie Linux. **Alotted time is 1Hr.** You may complete the tasks all in one sitting but are not required to do so. Time keeping is up to you. 
+
+Before you begin, read the entire list of tasks and conduct any preliminary research required to fully understand the goals. This should not be included in your 1 hour. Reach out to EJ Etherington at ej@reactiveops.com or @ejetherington on the Kubernetes Slack Workspace with any questions or clarifications you have.
```

Comment:
Just checking and I think it wouldn't be onerous to create a channel for each candidate and invite then as a guest to our slack. They'd have a direct line to us and I think it might be a good thing to do for all our interviewees, not just those taking the `tethys` version. Great idea @kimschles !